e2e/features/send-case-preHearing-bundle-failed.feature

```diff
@@ -0,0 +1,116 @@
+Feature: Generate hearing bundle
+
+
+  Background:
+    Given I am signed in as a `Legal Rep`
+    And I create a new case
+    And I save my initial appeal
+    And I submit my appeal
+    And I switch to be a `Case Officer`
+    And I request respondent evidence
+    And I upload respondent evidence
+    And I switch to be a `Legal Rep`
+    And I build my case
+    And I submit my case
+    And I upload additional evidence
+    And I switch to be a `Case Officer`
+    And I request respondent review
+    And I add the appeal response
+    And I request appellant review
+    And I request hearing requirements
+    And I switch to be a `Legal Rep`
+    And I submit hearing requirements with all yes
+    And I switch to be a `Case Officer`
+    And I record agreed hearing requirements yes path
+    And I switch to be a `Admin Officer`
+    And I list the case
+    And I switch to be a `Case Officer`
+    And I create case summary
+
+  @regression @RIA-2723
+  Scenario: Start decision and reasons after failed hearing bundle
+    When I select the `Send to preHearing` Next step
+    And I click the `Submit` button
+    Then I should see an alert confirming the case `has been updated with event: Send to preHearing`
+    Then I should only see the `caseOfficer_preHearing` case progress image
```

Comment:
Should possibly use 'And' instead of 'Then'

Reply:
Made the review change.

Reply:
test updates